lib/src/Base/Graph/openturns/Contour.hxx

```diff
@@ -130,6 +185,39 @@ private:
   /** Flag for drawing the labels */
   Bool drawLabels_;
 
+  /** Flag for filling the contours */
+  Bool isFilled_;
+
+  /** Color bar position */
+  String colorBarPosition_;
+
+  /** use of vmin value */
+  Bool isVminUsed_;
+
+  /** Min value of color map */
+  Scalar vmin_;
+
+  /** use of vmax value */
+  Bool isVmaxUsed_;
+
+  /** Max value of color map */
+  Scalar vmax_;
```

Comment:
it looks like the Interval object could replace all these 4 attributes

Reply:
I don't think so: vmin and vmax are optional and independent matplotlib arguments. How would you communicate that you want to set vmin, but not vmax ?

Reply:
these attributes can be initialized here in the .hxx instead of twice in the .cxx in each ctor